herbert_core/utilities/xml_io_tools/check_MATLAB_version.m

```diff
@@ -0,0 +1,15 @@
+function check_MATLAB_version()
+%CHECK_MATLAB_VERSION verifies if the MATLAB version is sufficient to run
+%xml_read/xml_write
+
+v = ver('MATLAB');
+vs = regexp(v.Version, '\d\d.\d','match','once'); % new Matlab after 2023a
+if isempty(vs)
+    vs = regexp(v.Version, '\d.\d','match','once'); % Matlab before 2023a
+end
```

Comment:
```suggestion
vs = regexp(v.Version, '\d\d?.\d','match','once'); 
```

Reply:
But why do it this way? There's a [`verlessthan`](https://uk.mathworks.com/help/matlab/ref/verlessthan.html) function. Why would we implement our own?

Reply:
if you click on this link, it will tell you at the top that verlessthan is deprecated, and gives you the replacement

Reply:
Deprecated as of 2020b, and we support back to 2018b

Reply:
It is still supported, i.e. will work, it is just deprecated. That means it may be removed in future releases, but is not yet.

Reply:
This is minor issue to fix third party code. I think what author did is good enough and further discussion is not worth time spend, so I am closing this issue.